flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/config/entries/ExecutionEntry.java

```diff
@@ -152,6 +156,8 @@ protected void validate(DescriptorProperties properties) {
 		properties.validateInt(EXECUTION_RESTART_STRATEGY_MAX_FAILURES_PER_INTERVAL, true, 1);
 		properties.validateString(EXECUTION_CURRENT_CATALOG, true, 1);
 		properties.validateString(EXECUTION_CURRENT_DATABASE, true, 1);
+		properties.validateBoolean(EXECUTION_INTERACTIVE_VIEW_ENABLED, true);
+		properties.validateInt(EXECUTION_MAX_QUERY_TIMEOUT_MS, true, -1);
```

Comment:
nit: feels like we can set a default timeout when EXECUTION_INTERACTIVE_VIEW_ENABLED is false?